src/network/networkprotocol.h

```diff
@@ -922,6 +922,8 @@ enum ToServerCommand
 		[2+12+12+4+4] u32 keyPressed
 		[2+12+12+4+4+1] u8 fov*80
 		[2+12+12+4+4+4+1] u8 ceil(wanted_range / MAP_BLOCKSIZE)
+		[2+12+12+4+4+4+1+1] u8 camera_inverted (bool)
```

Comment:
```suggestion
		[2+12+12+4+4+4+1+1+1] u8 camera_inverted (bool)
```

Reply:
Same as above, are we missing an entry in between?

Reply:
Wait, I added exactly 1 byte. Where is this wrong?

Reply:
Oops, yes. 923 was wrong. You've fixed that now I'm seeing.